index.html

```diff
@@ -13,7 +13,7 @@
 
 <h2 style="margin-top:20px; margin-bottom:20px">{{ site.url }}</h2>
 
-<p>Welcome to NIPY. The NIPY community is a community of practice devoted to the use of the Python programming language in the analysis of neuroimaging data. You can find us on <a href="https://github.com/nipy" target="_blank">github</a>, as well as <a href="http://neuroimaging.tumblr.com" target="_blank">social media</a>. We welcome <a href="{{ '/contribute.html' | prepend: site.baseurl">contributions</a> and welcome you to read about our <a href="{{ '/conduct.html' | prepend: site.baseurl }}">standards of conduct</a>, or <a href="{{ '/help.html' | prepend: site.baseurl }}">ask for help.</a> We develop the following projects:</p>
+<p>Welcome to NIPY. We are a community of practice devoted to the use of the Python programming language in the analysis of neuroimaging data. You can find us on <a href="https://github.com/nipy" target="_blank">github</a>, as well as <a href="http://neuroimaging.tumblr.com" target="_blank">social media</a>. We welcome <a href="{{ '/contribute.html' | prepend: site.baseurl">contributions</a> and welcome you to read about our <a href="{{ '/conduct.html' | prepend: site.baseurl }}">standards of conduct</a>, or <a href="{{ '/help.html' | prepend: site.baseurl }}">ask for help.</a> We develop the following projects:</p>
```

Comment:
I think it's a neatly matching description:
https://en.wikipedia.org/wiki/Community_of_practice

On Sat, May 7, 2016 at 10:30 AM, Satrajit Ghosh <notifications@github.com>
wrote:

> In index.html
> <https://github.com/nipy/nipy.github.com/pull/34#discussion_r62419847>:
>
> > @@ -13,7 +13,7 @@
> >
> >  <h2 style="margin-top:20px; margin-bottom:20px">{{ site.url }}</h2>
> >
> > -<p>Welcome to NIPY. The NIPY community is a community of practice devoted to the use of the Python programming language in the analysis of neuroimaging data. You can find us on <a href="https://github.com/nipy" target="_blank">github</a>, as well as <a href="http://neuroimaging.tumblr.com" target="_blank">social media</a>. We welcome <a href="{{ '/contribute.html' | prepend: site.baseurl">contributions</a> and welcome you to read about our <a href="{{ '/conduct.html' | prepend: site.baseurl }}">standards of conduct</a>, or <a href="{{ '/help.html' | prepend: site.baseurl }}">ask for help.</a> We develop the following projects:</p>
> > +<p>Welcome to NIPY. We are a community of practice devoted to the use of the Python programming language in the analysis of neuroimaging data. You can find us on <a href="https://github.com/nipy" target="_blank">github</a>, as well as <a href="http://neuroimaging.tumblr.com" target="_blank">social media</a>. We welcome <a href="{{ '/contribute.html' | prepend: site.baseurl">contributions</a> and welcome you to read about our <a href="{{ '/conduct.html' | prepend: site.baseurl }}">standards of conduct</a>, or <a href="{{ '/help.html' | prepend: site.baseurl }}">ask for help.</a> We develop the following projects:</p>
>
> delete: "of practice"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/nipy.github.com/pull/34/files/707efb67d170739e443f33b0d910fea1dfcb91e4#r62419847>
>